todo-app/src/reducers/reducer.js

```diff
@@ -0,0 +1,57 @@
+const todoList = [
+  { id: 1, name: "sweeping", completed: false },
+  { id: 2, name: "mopping", completed: false },
+  { id: 3, name: "washing dishes", completed: false }
+];
+
+export const initialState = {
+  todoList,
+  todoValue: ""
+  // etc
+};
+
+// const MARK_TODO = 'MARK_TODO'; // we will use the same to "complete" and "uncomplete"
+// // etc
+
+export const reducer = (state, action) => {
+  switch (action.type) {
+    case "MARK_TODO":
```

Comment:
Via convention, line 18 should be a variable pointing to the string of a similar name.